### PR TITLE
Add .rc1 to katello-repos spec file

### PR DESCRIPTION
**Pull request by zjhuntin**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by zjhuntin** [comment]
[test]

File: katello/katello-repos/katello-repos.spec [hunk 1]
```diff
@@ -1,3 +1,7 @@
+%global prever .rc1
+%global pulp_release beta
```
Comment (ehelms) [inline katello/katello-repos/katello-repos.spec]: This is now stable

**Comment by zjhuntin** [comment]
[test]

**Comment by ekohl** [comment]
[test packaging]

**Comment by ekohl** [comment]
This needs a look at why the build is failing.

**Comment by ehelms** [comment]
/me facepalms

I totally missed this in the 3.6 RC1 process and is why the repos RPM is so messed up.

**Comment by ehelms** [comment]
[test packaging]

**Comment by ehelms** [comment]
[test packaging]

**Comment by ehelms** [comment]
Woot working now! Time to fix 3.6

**Pull request merged**